lib/fastlane/actions/actions_helper.rb

```diff
@@ -83,7 +83,7 @@ def self.load_default_actions
     end
 
     def self.load_external_actions(path)
-      raise 'You need to pass a valid path' unless File.exist?(path)
+      fail 'You need to pass a valid path' unless File.exist?(path)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.